docs/contributing/code-contributions.md

```diff
@@ -119,6 +120,22 @@ Using Docker Compose, you can start and stop a WordPress instance and integrate
 
 Check [Debugging the build process](/docs/debugging-the-build-process/) page to learn how to debug Gatsby.
 
+## Official theme development
+
+This section is for official theme development in Gatsby's monorepo. If you are looking
+to build your own theme, see [building themes](/docs/themes/building-themes/).
+
+Before getting started, make sure that you have
+[set up your local dev environment](/contributing/setting-up-your-local-dev-environment/)
+and that you're on the latest version of `gatsby-dev-cli`.
+
+- Scaffold out a starter that uses the theme you want to work on:
```

Comment:
Where should this happen? Are we talking about the starter theme in the starter repo?
Based on my testing that seems correct. So perhaps we should make clear that relative path is meant to be whatever starter that runs the theme is?

Reply:
On closer look it seems like you're saying that a starter using the theme needs to exist, otherwise make one. Then create a site based on that starter. Is that correct? If so, we should be explicit.

Reply:
I realized we can simplify this by using monorepo starters, so I've updated the instructions to reflect that. Let me know if that makes things more clear!

Reply:
Haha, that's what I did without realizing that wasn't what you meant. I feel vindicated!